integration-tests/run.sh

```diff
@@ -5,7 +5,9 @@
 set -euo pipefail
 
 # This deliberately can be run outside the container - sometimes you want to test on
-# the host
+# the host. TODO this is currently failing, unless you specify some env vars
+# explicitly. Here's a sample command to get it working for now:
+# DARK_CONFIG_APISERVER_HOST=darklang.localhost:9000 DARK_CONFIG_BWDSERVER_HOST=builtwithdark.localhost:11001 ./integration-tests/run.sh
```

Comment:
better than nothing, until #3810 is resolved.

Reply:
I know I keep looking this up in Slack history, anyway

Reply:
Yeah, this is a good idea.